docs/src/operate/registration/index.md

```diff
@@ -1,11 +1,21 @@
 ---
-title: Registration
+title: Entity Management
 tags: [Registration, Deregistration]
 sidebar_position: 11
 ---
 
 import DocCardList from '@theme/DocCardList';
 
-Managing registration and deregistration of child devices and services on %%te%%.
+%%te%% provides two different API interfaces for managing entities on a device:
```

Comment:
This introduction is missing something. The focus is on HTTP vs MQTT but the purpose of these APIs is a bit vague. I would give a short overview of these *core functionality*.

```suggestion
%%te%% provides an API for managing entities on a device: to register child devices and services with metadata describing their role and relationship.

%%te%% provides two different flavor of this API:
```

Reply:
Revamped this intro significantly in 4ff49e7, highlighting the pros and cons of both APIs